lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -65,25 +60,39 @@ class DSBottomSheetService {
     return Container(
       height: 30.0,
       decoration: _border(),
-      child: Row(
-        mainAxisSize: MainAxisSize.max,
-        mainAxisAlignment: MainAxisAlignment.center,
+      child: Column(
         children: [
-          Container(
-            height: 4.0,
-            width: 32.0,
-            decoration: const BoxDecoration(
-              color: DSColors.neutralMediumWave,
-              borderRadius: BorderRadius.all(
-                Radius.circular(15.0),
+          Row(
+            mainAxisSize: MainAxisSize.max,
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Container(
+                height: 4.0,
+                width: 32.0,
+                decoration: const BoxDecoration(
+                  color: DSColors.neutralMediumWave,
+                  borderRadius: BorderRadius.all(
+                    Radius.circular(15.0),
+                  ),
+                ),
               ),
-            ),
+            ],
           ),
+          _appBar(padding);
```

Comment:
why do you create this _appBar here?  
it cannot be passed in the bottomSheet child parameter?

Reply:
The `padding` in parameter is not valid